build/ansible/roles/pmm2-images/tasks/main.yml

```diff
@@ -100,7 +100,7 @@
         non_unique: true
       loop:
         - { name: pmm, gid: 1000 }
-        - { name: nginx, gid: 999 }
+        - { name: nginx, gid: 1002 }
```

Comment:
let's remove this user and run nginx under pmm user to not have issues with having multiple users in future

Reply:
Done.